src/main/kotlin/org/vineflower/ijplugin/VineflowerSettingsPanel.kt

```diff
@@ -42,7 +32,7 @@ class VineflowerSettingsPanel(var prevVineflowerVersion: SemVer?) {
 
     init {
         pluginSettingsPanel.border = IdeBorderFactory.createTitledBorder("Plugin Settings")
-        vineflowerSettingsPanel.border = IdeBorderFactory.createTitledBorder("Vineflower Settings")
+//        vineflowerSettingsPanel.border = IdeBorderFactory.createTitledBorder("Vineflower Settings")
```

Comment:
You can delete this line, no need to leave it commented out as we have git history